validator_test.go

```diff
@@ -632,7 +636,11 @@ func TestValidateRequiredFailing(t *testing.T) {
 
 			t.Logf("Unpack returned error: %v", err)
 			err = err.(Error).Reason()
-			assert.Equal(t, test.err, err)
+			logTmpl := "expected:%q got:%q"
```

Comment:
I don't think we always need to log. Appropos want vs. got, I like to use go-cmp in my own projects. Asserting becomes this then:

```
if diff := cmp.Diff(want, got); diff != "" {
	t.Errorf("mismatch (-want +got):\n%s", diff)
}
```

Reply:
Non-error test logging only happens when `go test` is given the `-v` option, does that change your concern at all?

Reply:
We run most tests with `-v` and the output is quite verbose already. Just noticed that the go-ucfg travis file does not use `-c`, but I will add it to be consistent with other projects (need to update the travis file because I'm adding go modules support anyways).

Reply:
Removed.